### PR TITLE
fix(settings): alert message style improvements

### PR DESCRIPTION
**Pull request by Chrissy**
We have gotten feedback that `AlertMessage` is very large. 

This is particularly problematic in instances like the one below, where other UI is going on. The giant text becomes the most important thing on the page.

image

That said, i kinda think `AlertMessage` is *always* too big. It is just an empty message, after all. It shouldn't be bigger than the header of the page.

This pull gives us two size options: 16px and 18px. It uses em units to adjust the size of the margin, avatar, etc. because i still secretly like em units and they are useful here.

image

image

image

this also fixes up some places where alert message is either not used or used improperly:

image

image

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/theme.jsx [hunk 1]
```diff
@@ -87,6 +87,9 @@ const theme = {
   },
 
   grid: 8,
+  fontSizeSmall: '12px',
+  fontSizeMedium: '16px',
+  fontSizeLarge: '18px',
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/utils/theme.jsx]: these are maybe not ideal but it's better than nothing, and as of today calling stuff nested inside the text object will not pass tests.

Comment (ckj) [reply src/sentry/static/sentry/app/utils/theme.jsx]: Agreed. 👍on having them here for now.

**Review by ckj — APPROVED** [review]
+1. Fwiw they were supposed to be the headline and not body copy, but even then they were still too big.

P.s. I think you mean _Empty_ message and not Alert Message. I stared at the Alert Messages in the screenshot for a solid minute before I realized they weren't what was being changed 😛

**Comment by Chrissy** [comment]
@ckj ugh you are totally right i meant `EmptyMessage` I think i'm getting component vertigo!

**Pull request merged**